lib/puma/dsl.rb

```diff
@@ -588,6 +592,17 @@ def after_worker_fork(&block)
       @options[:after_worker_fork] << block
     end
 
+    # Code to run after puma is booted in a single mode
+    #
+    # @note Single mode only.
```

Comment:
Why this would be Single mode only? Cluster mode also calls `fire(:on_booted)`

Reply:
Johnny is correct:

https://github.com/puma/puma/blob/5f255fcd132a3d67608a3d82b31c540bdbf69bcf/lib/puma/cluster.rb#L468 https://github.com/puma/puma/blob/5f255fcd132a3d67608a3d82b31c540bdbf69bcf/lib/puma/cluster.rb#L477

I guess the docs was written like this because the request (#2635, https://github.com/puma/puma/issues/1230#issuecomment-839953916) came from a user using single mode where `after_worker_boot` (aliased to `after_worker_fork`) doesn't exist. https://github.com/puma/puma/blob/5f255fcd132a3d67608a3d82b31c540bdbf69bcf/lib/puma/cluster.rb#L104

Reply:
Thanks for catching this, correct it runs for both, clustered and single mode, I've changed the description. The question is, what would be the expected behavior? At the end the name of the event is `on_booted` so I think it makes sense to be run on both: single and clutered. Any thoughts @nateberkopec ?